tests/model/test_experiment_list.py

```diff
@@ -924,3 +925,34 @@ def test_experimentlist_imagesequence_decode(mocker):
     assert len(experiments2.scans()) == 3
     for expt in experiments2:
         assert expt.imageset is experiments2.imagesets()[0]
+
+
+def test_experiment_list_change_basis(dials_data):
```

Comment:
```suggestion
def test_experimentlist_change_basis(dials_data):
```
just consistency-OCD